chain/neutrino.go

```diff
@@ -107,14 +107,12 @@ func (s *NeutrinoClient) Start() error {
 	s.clientMtx.Lock()
 	defer s.clientMtx.Unlock()
 	if !s.started {
-		// Reset the client state.
```

Comment:
Why are we removing those comments again?

Reply:
I added these earlier when fixing the race condition in start/stop and removed them in this commit just to make the diff against master smaller, wasn't sure if they were useful enough to be kept.  I can add them back. 👍🏾